libraries/chain/db_maint.cpp

```diff
@@ -830,7 +830,7 @@ void cleanup_invalid_feeds_hf_868( database& db )
       // you can remove this double, and the check_call_orders()
       // below that relies on it. HF343 will take care of the issue
       // and there is no need for the extra check_call_orders() call
-      double old_price = bitasset_data.settlement_price.to_real();
+      const auto& old_price = bitasset_data.settlement_price;
```

Comment:
Need a copy here, otherwise unable to check if it's changed.

Reply:
Oops! So worried about not making copies I didn't make a copy when I needed to!